python/test/unit/hopper/test_persistent_warp_specialized_gemm.py

```diff
@@ -139,26 +143,13 @@ def test_user_defined_persistent_non_warp_specialized_gemm(M, N, K, BLOCK_M, BLO
     num_SMs = torch.cuda.get_device_properties('cuda').multi_processor_count
     grid = lambda META: (num_SMs,)
 
-    def call_vintage():
+    if KERN == 'vintage':
```

Comment:
Maybe we can use `USE_TMA = True / False`? I'm not sure, just a suggestion. What do you think?

Reply:
Yeah, it's make sense. Thanks.

Reply:
done